source/_components/camera.markdown

```diff
@@ -45,6 +45,16 @@ Take a snapshot from a camera.
 
 The path part of `filename` must be an entry in the `whitelist_external_dirs` in your [`homeassistant:`](/docs/configuration/basic/) section of your `configuration.yaml` file.
 
+For example, the following action in an automation would take a snapshot from "yourcamera" and save it to /tmp with a timestamped filename.
+
+```yaml
+action:
+  service: camera.snapshot
+  data:
+    entity_id: camera.yourcamera
+    filename: '/tmp/yourcamera.{{ as_timestamp(now()) }}.jpg'
```

Comment:
This would create a file like `yourcamera.1512579138.00433.jpg`, which is hardly useful. Can you simplify the filename so that it is more user-friendly?

Reply:
I hear you, but that's literally the filename I wanted. Being a UNIX guy, I'm used to epoch timestamps... And it's sortable! But I'm a weirdo.

How about we change "as_timestamp(now())" to "now().strftime("%Y%m%d-%H%M%S")" which is a more human-readable format?

Reply:
That looks good :+1: